config/filter.d/ignorecommands/apache-fakegooglebot.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Based on: https://isc.sans.edu/forums/diary/When+Google+isnt+Google/15968/
```

Comment:
Worth noting I like the idea of doing it all in python, the thought of an external command kept me from seeing that as a better solution.